contentpal/src/test/java/org/dmfs/android/contentpal/utils/ContentOperationBuilderCountMatcher.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.android.contentpal.utils;
+
+import android.content.ContentProviderOperation;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+
+import java.lang.reflect.Field;
+import java.util.Locale;
+
+
+/**
+ * A {@link ContentProviderOperation.Builder} {@link Matcher} which matches the operation type.
+ * <p>
+ * TODO: match Uri
+ *
+ * @author Marten Gajda
+ */
+public final class ContentOperationBuilderCountMatcher extends TypeSafeDiagnosingMatcher<ContentProviderOperation.Builder>
```

Comment:
Could 'expected` added to the name as well? It was misleading for me, I thought it is referring to the number of values.
I know it will be even longer but the few times we need to read it, it may help I think.

Reply:
Sure.